desktop-exporter/app/main.jsx

```diff
@@ -9,4 +9,9 @@ function App() {
   );
 }
 
-ReactDOM.render(<App />, document.querySelector('#root'));
+// ReactDOM.render is no longer supported in React 18 - using createRoot instead
```

Comment:
This is good context to have for the PR, but I don't think this comment should live in the code forever.

Reply:
Is gone now!